### DISCUSSION TITLE
pve-bridge fails with status 512

### DISCUSSION THREAD

**Discussion by issmirnov** [tteck/Proxmox#148] [General]
Hello,

I found the guide at https://www.inspectmygadgets.com/set-up-a-home-assistant-vm-on-proxmox/ which pointed me to this repo.

I am running proxmox via KVM on an Ubuntu 22.04 machine, and trying to install hassos. My command is:

```bash
bash -c "$(wget -qLO - https://raw.githubusercontent.com/tteck/Proxmox/main/vm/haos_vm.sh)"
```

I'm getting the following error output:

```
[INFO] Using local-lvm for Storage Location.
[INFO] Container ID is 101.
 Getting URL for Latest Home Assistant Disk Image... ✔ 
 Downloaded qcow2 Disk Image... ✔
 Extracting Disk Image... ✔ 
 Creating HAOS VM... ✔ 
 Adding Serial Port and Configuring Console... ✔ 
 Starting Home Assistant OS VM... failed to open /proc/sys/net/ipv6/conf/tap101i0/disable_ipv6 for writing: No such file or directory
kvm: -netdev type=tap,id=net0,ifname=tap101i0,script=/var/lib/qemu-server/pve-bridge,downscript=/var/lib/qemu-server/pve-bridgedown,vhost=on: network script /var/lib/qemu-server/pve-bridge failed with status 512
start failed: QEMU exited with code 1
[ERROR] 255@192 Unknown failure occured.
  Logical volume "vm-101-disk-1" successfully removed
  Logical volume "vm-101-disk-0" successfully removed
bash -c   26.34s user 4.75s system 44% cpu 1:10.35 total
```

Content of `/etc/network/interfaces` on the proxmox host is:

```
auto lo
iface lo inet loopback

iface enp1s0 inet manual

auto vmbr0
iface vmbr0 inet static
        address 10.3.37.48/28
        gateway 10.3.37.1
        bridge-ports enp1s0
        bridge-stp off
        bridge-fd 0
```

Do you have any thoughts on what I could tweak to get this to work? The proxmox install is basically vanilla, no weird customization.

It looks like there's some problem with the network initialization, but I'm not familiar enough with proxmox to debug it on the fly.

Thank you in advance for your advice!

**Comment by tteck** [comment]
I honestly have no idea where to start with that setup. I've never ran Proxmox that way. Sorry.
I'll move it over to Discussions maybe someone can help
